client/app/services/query-result.js

```diff
@@ -523,10 +523,16 @@ function QueryResultService($resource, $timeout, $q, QueryResultError) {
       return `${queryName.replace(/ /g, '_') + moment(this.getUpdatedAt()).format('_YYYY_MM_DD')}.${fileType}`;
     }
 
-    static get(dataSourceId, query, maxAge, queryId) {
+    static get(dataSourceId, parameters, query, maxAge, queryId) {
```

Comment:
I'm curious, why have the parameters before the query?

Reply:
Yeah, found that peculiar ordinality [here](https://github.com/getredash/redash/blob/master/redash/handlers/query_results.py#L30) and thought I'd stay consistent.

Reply:
No reason to be consistent with that abomination. Once you're done , we will be able to finally get rid of it. 

![](https://media.giphy.com/media/5nsiFjdgylfK3csZ5T/giphy.gif)